### DISCUSSION TITLE
What'd be likely to happen if was to try downgrading from 2021.03.06?

### DISCUSSION THREAD

**Discussion by Madddcow** [ludwig-v/wireless-carplay-dongle-reverse-engineering#39] [Q&A]
As per the subject, what'd be likely to happen if was to try downgrading from 2021.03.06? Would the downgrade fail or would it brick the device?  Just curious.

**Comment by ludwig-v** [comment]
It just won't work

**Reply by Madddcow** [reply]
Ok.  Thanks!

**Reply by ghost** [reply]
I was able to downgrade by using the roll back feature in Web Settings on the devie, after I rolled back, I then upgraded and installed the Custom.

However, After the downgrade I was no longer to roll back because the Roll Back option disappeared from settings.
